packages/react-components/src/templates/NetworkPageHeader.tsx

```diff
@@ -29,7 +29,7 @@ const controlsStyles = css({
   gridColumnGap: `${18 / perRem}em`,
   alignItems: 'center',
   paddingTop: `${18 / perRem}em`,
-  [`@media (min-width: ${smallDesktopScreen.min}px)`]: {
+  [`@media (min-width: ${smallDesktopScreen.min + 1}px)`]: {
```

Comment:
`min-width` being one greater than `smallDesktop`s `min` doesn't seem to make sense to me - previously it applied to small desktop regardless of orientation, now it only just misses portrait small desktop but hits landscape small desktop. Perhaps the intention was to make it miss tablets regardless of orientation? If so,
```suggestion
  [`@media (min-width: ${tabletScreen.max + 1}px)`]: {
```
(needs import)